charts/sn-platform/values.yaml

```diff
@@ -79,7 +79,7 @@ components:
   # bookkeeper
   bookkeeper: true
   # bookkeeper - autorecovery
-  autorecovery: false
+  autorecovery: true
```

Comment:
Does this configuration need to be enabled by default?

Reply:
I wonder why it wasn't enabled by default as it is used in ledger recovery and scaling down of the cluster, and I just turned it on. @ericsyh What's your opinion?

Reply:
I think it will be better to turn the autorecovery on by default cause some users (especially new to pulsar)may not understand the usage of autorecovery. It will increase the risk of data lose for these users when they scaling down the bk or unable to create new bk pod in some cases.